src/test/java/com/example/universe/simulator/entityservice/integration/GalaxyIntegrationTest.java

```diff
@@ -83,6 +91,15 @@ void test() throws Exception {
             .contentType(MediaType.APPLICATION_JSON)
             .content(objectMapper.writeValueAsString(dto))
         );
+
+        //check if resultDto got updated in cache
+        GalaxyDto fromCacheUpdated = modelMapper.map(
+                objectMapper.convertValue(redisTemplate.opsForValue().getAndDelete("Galaxy::" + addedDto1.getId()), Galaxy.class),
```

Comment:
the same

Reply:
Same answer

Reply:
If unnecessary conversion is meant, will modify